### PR TITLE
Cover pkg/kexecbin

### PR DESCRIPTION
**Pull request by walterchris**
Signed-off-by: Christian Walter <christian.walter@9elements.com>

### CODE REVIEW

**Review by depate — DISMISSED** [review]
Deprecate func Equal

File: pkg/kexecbin/kexecbin.go [hunk 1]
```diff
@@ -55,6 +51,19 @@ func KexecBin(kernelFilePath string, kernelCommandline string, initrdFilePath st
...
 
+	return loadCommands
+}
+
+// KexecBin uses kexec-tools binary and runtime architecture detection
+// to execute abritary files.
+func KexecBin(kernelFilePath string, kernelCommandline string, initrdFilePath string, dtFilePath string) error {
+	baseCmd, err := exec.LookPath("kexecbin")
+	if err != nil {
+		return err
```
Comment (ChriMarMe) [inline pkg/kexecbin/kexecbin.go:62]: Maybe you can also cover this by making the function exec.LookPath a global variable and write a dummy for the test?

**Review by llogen — COMMENTED** [review]
Nothing to add.

**Comment by ChriMarMe** [comment]
Das Package wird nirgends benutzt. Vielleicht waere Loeschen sinnvoller.

**Comment by ChriMarMe** [comment]
I opened a PR to remove the package from u-root. Therefor I close this PR

**Pull request closed without merging**